### PR TITLE
Delete option added for favorite project

### PR DESCRIPTION
**Pull request by devux**
I just added delete option for favorite project in the user dashboard by that user can able to remove favorite project from their list

### CODE REVIEW

File: app/views/home/dashboard/_projects.html.erb [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
+        
       <% @favorite_projects.each do |project| %>
+        <tr>
+        <td>
         <li><%= link_to project.name, project_path(project) %></li>
+        </td>
+        <td>
+        <%= link_to 'Remove', favorite_path(id: project.id), method: :delete, data: { confirm: 'Are you sure?'}, class: 'favorite radius button'  %>
+        </td>
+        </tr>
```
Comment (steverob) [inline app/views/home/dashboard/_projects.html.erb]: @devakdo proper indentation please :)

Comment (devakdo) [inline app/views/home/dashboard/_projects.html.erb]: ok let me do it

**Comment by steverob** [comment]
@devakdo can provide a screenshot as well please? Thanks!

**Comment by devakdo** [comment]
do i want to add any other things to complete this feature

**Comment by iamBalaji** [comment]
why not this pull request get accepted?

**Comment by steverob** [comment]
@devakdo can you fix the @houndci comments, attach a screenshot, squash the commits please. Will help the maintainers :+1:

**Comment by devakdo** [comment]
Ok @steve let me do it

**Comment by courte** [comment]
@devakdo, thanks for taking this on! :+1: :smiley: 

Instead of deleting FavoriteProjects outright, we'd prefer to hold on to that data. For example, a project being unfavorited a number of times might be a sign that it is no longer being maintained or that it's difficult to install - important information for us to have. Meanwhile overall FavoriteProjects (deleted or not) demonstrate the overall initial appeal of a project. If we delete them, this kind of info is gone forever.

Holding onto "deleted" FavoriteProjects would mean adjusting the schema, adding a scope to the FavoriteProject, and changing what displays in the view. Would you be open to taking that work on?

**Pull request closed without merging**